+replab/+nu/symIrrepImages.m

```diff
@@ -57,11 +57,11 @@
     
 elseif isequal(partition,[1 1]) %sign
     SB = [-1];
-    SA = [1];
```

Comment:
@sereneshum Both a and b represent the same generator for S(2), right?

Reply:
Yeah, they actually should be the same. Sorry about that